test/gpu/native/studies/transpose/transpose.chpl

```diff
@@ -70,9 +110,11 @@ on here.gpus[0] {
   var timer: stopwatch;
   for 1..#numTrials {
     timer.start();
-    if useNaive {
+    if impl == naive {
```

Comment:
I assume Chapel select statements will error out if we added a new case and forgot to add it here. Admittedly is an unlikely thing to forget to if you were updating this test, but assuming Chapel does that check then I think it's a best practice to use `select` whenever doing something that should cover all the cases of an enum.